sphinx_rtd_theme/layout.html

```diff
@@ -24,30 +23,18 @@
   {%- endblock -%}
 
   {#- CSS #}
-  {%- if sphinx_version_info < (4, 0) -%}
-    <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
-    <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
-  {%- endif %}
-  {%- for css_file in css_files %}
-    {%- if css_file|attr("filename") %}
-      {{ css_tag(css_file) }}
-    {%- else %}
-      <link rel="stylesheet" href="{{ pathto(css_file, 1)|escape }}" type="text/css" />
-    {%- endif %}
+  {%- for css in css_files %}
+    {{ css_tag(css) }}
```

Comment:
Please keep the old logic for `css_files`. See #1537 for the rationale.

Reply:
Oh, thanks for pointing this out. I was making the same mistake again 😅

Reply:
🤦🏼 -- I'll come back to this another day because it seems I'm pretty distracted today and I can't focus clearly. Thanks again for your time reviewing this 🙏🏼

Reply:
Done!

Reply:
Thank you. LGTM now.